tools/SetupFlow/DevHome.SetupFlow.Common/TelemetryEvents/ConfigurationUnitResultEvent.cs

```diff
@@ -15,7 +15,7 @@ internal sealed class ConfigurationUnitResultEvent : EventBase
 {
     private readonly ApplyConfigurationUnitResult _unitResult;
 
-    public string UnitName => _unitResult.Unit.UnitName;
+    public string UnitName => _unitResult.Unit.Type;

```

Comment:
NIT: Should this variable still be called UnitName?

Reply:
`UnitName` was renamed to `Type` in this [commit](https://github.com/microsoft/winget-cli/commit/721c81ff47c12c454c4e8d973701b7899f2f22ef#diff-079d03a1599653ae5a9de8e2159c9573b169abb50f75012628f2ea3fa6f2adb0L49-R62)

Reply:
Renamed `UnitName` to `Type` to align with the changes. Thank you!